v03_pipeline/lib/methods/relatedness.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
+
+import hail as hl
+from gnomad.sample_qc.pipeline import filter_rows_for_qc
+
+from v03_pipeline.lib.model import ReferenceGenome
+
+
+# Stub port from https://github.com/broadinstitute/gnomad_qc/blob/9763d8a581665ed0be8bf2ea09d74e3cabc4806e/gnomad_qc/v2/resources/sample_qc.py#L40
+def qc_mt_path(reference_genome: ReferenceGenome) -> str:
```

Comment:
I hardcoded these to avoid trying to import `gnomad_qc`.

Reply:
why would we avoid importing gnomad_qc ?

Reply:
why would we want to?

I think just dependency weight (how the complexity of the dependency weighs against re-implementing the functionality locally).  We need these to run on `dataproc`, which means we need clean `pip` installs or a bundled `.whl` somewhere.  I found [gnomad](https://pypi.org/project/gnomad/) but not `gnomad_qc` on pip.

Reply:
I agree that we do not want to be importing a heavyweight module just to get us a couple lightweight helper functions.